.travis.yml

```diff
@@ -1,5 +1,4 @@
-# opt-in to Travis new infrastructure
-sudo: false
+sudo: true
```

Comment:
we can't even `chmod +x` things we just downloaded ourselves in our own home directory? that surprises me, I wouldn't expect superuser privileges to be necessary. are you sure?